src/object-field/object-field.component.ts

```diff
@@ -49,6 +49,9 @@ export class ObjectFieldComponent extends AbstractFieldComponent {
     public changeDetectorRef: ChangeDetectorRef,
     public keysStoreService: KeysStoreService) {
     super(appGlobalsService, pathUtilService, changeDetectorRef);
+    this.appGlobalsService.adminModeSubject.subscribe(adminMode => {
```

Comment:
Is this needed? Since you already markForCheck in primitive-field-component (which is a child of this component) this should be checked as well.